tests/system/lab/remove

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+time sudo apt remove -y foglamp foglamp-gui # better to purge?
```

Comment:
I would tend to use `purge` if you do not want to keep any configuration files around.